### PR TITLE
refactor: Convert Banner stories to csf format

### PR DESCRIPTION
**Pull request by brianacnguyen**
## **Description**
- Convert Banner Alert story to csf format
- Convert Banner Tip story to csf format
- Convert Banner story to csf format

## **Screenshots/Recordings**
Simulator Screen Shot - iPhone 14 Pro - 2023-09-25 at 21 16 30
Simulator Screen Shot - iPhone 14 Pro - 2023-09-25 at 21 16 36
Simulator Screen Shot - iPhone 14 Pro - 2023-09-25 at 21 16 38
Simulator Screen Shot - iPhone 14 Pro - 2023-09-25 at 21 16 43

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained:
  - [ ] What problem this PR is solving.
  - [ ] How this problem was solved.
  - [ ] How reviewers can test my changes.
- [ ] I’ve indicated what issue this PR is linked to: Fixes #???
- [ ] I’ve included tests if applicable.
- [ ] I’ve documented any added code.
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   stories: [
     '../app/component-library/components/Cards/Card/**/*.stories.?(ts|tsx|js|jsx)',
-    '../app/component-library/components/Banners/Banner/variants/BannerAlert/**/*.stories.?(ts|tsx|js|jsx)',
+    '../app/component-library/components/Banners/**/*.stories.?(ts|tsx|js|jsx)',
```
Comment (georgewrmarshall) [inline .storybook/main.js:4]: Could we simplify this file to just all component library folders and files?

File: app/component-library/components/Banners/Banner/variants/BannerTip/BannerTip.stories.tsx [hunk 2]
```diff
@@ -1,44 +1,32 @@
...
+  },
+};
+export default BannerTipMeta;
 
-  return {
-    logoType: logoTypeSelector,
-    title,
-    description,
+export const BannerTip = {
+  args: {
```
Comment (georgewrmarshall) [inline app/component-library/components/Banners/Banner/variants/BannerTip/BannerTip.stories.tsx:29]: Would it make sense to set the default for `logoType` in the controls it's undefined

Screenshot 2023-10-12 at 12 33 33 PM

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! A couple non-blocking comments that might be worth investigation. 
- pulled branch checked stories and controls

When I edit the `actionButtonProps.variant` controls it breaks storybook


https://github.com/MetaMask/metamask-mobile/assets/8112138/59a2405b-0589-46c4-8ba2-ee31657ddd2a

Console error to be aware of

Screenshot 2023-10-12 at 12 35 02 PM

**Comment by brianacnguyen** [comment]
> LGTM! A couple non-blocking comments that might be worth investigation.
> 
> * pulled branch checked stories and controls
> 
> When I edit the `actionButtonProps.variant` controls it breaks storybook
> 
>  error.mov 
> Console error to be aware of
> 
> Screenshot 2023-10-12 at 12 35 02 PM

fixed

**Review by georgewrmarshall — DISMISSED** [review]

**Review by georgewrmarshall — APPROVED** [review]

**Pull request merged**